### DISCUSSION TITLE
FLV player

### DISCUSSION THREAD

**Discussion by wangstar** [ruffle-rs/ruffle#5931] [Q&A] [answered]
`<embed src="flvplayer.swf" FlashVars="flv=test.flv" type="application/x-shockwave-flash"> </embed>`

**Comment by n0samu** [comment, marked as answer]
I recommend converting the video to MP4 and using the [HTML5 video tag](https://www.w3schools.com/html/html5_video.asp).

**Reply by wangstar** [reply]
`yes.` In my new project I recommend converting the video to MP4 and using the HTML5 video tag.
**But**. When I use ruffle-extension to test &lt;embed src=&quot;flvplayer.swf&quot; FlashVars=&quot;flv=test.flv&quot; type=&quot;application/x-shockwave-flash&quot;&gt; &lt;/embed&gt;, it will be replaced by ruffle -embed。

**Reply by n0samu** [reply]
If you're the owner of the site, you can use the `"polyfills": false` option to prevent Ruffle from replacing the embed tag: https://github.com/ruffle-rs/ruffle/wiki/Using-Ruffle#configuration-options
You can also turn off the Ruffle extension when visiting the site. Click the extension icon and turn off the "Play Flash content in Ruffle" switch.

**Reply by wangstar** [reply]
Thank you for your answer! I have replaced it with video-flv.
